apps/ui/src/pages/applications/sections/ethics.tsx

```diff
@@ -18,72 +18,138 @@
  */
 
 import { UploadOutlined } from '@ant-design/icons';
-import { EthicsFileEnum, ethicsSchema, type EthicsSchemaType } from '@pcgl-daco/validation';
-import { Button, Flex, Form, notification, theme, Typography, Upload, UploadProps } from 'antd';
+import { ethicsSchema, type EthicsSchemaType } from '@pcgl-daco/validation';
+import { Button, Flex, Form, notification, theme, Typography, Upload, UploadFile } from 'antd';
 import { createSchemaFieldRule } from 'antd-zod';
+import { RcFile, UploadChangeParam } from 'antd/es/upload';
+import { useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import { useOutletContext } from 'react-router';
 
+import useEditApplication from '@/api/mutations/useEditApplication';
+import useGetDownload from '@/api/queries/useGetDownload';
+import useGetFile from '@/api/queries/useGetFile';
 import SectionWrapper from '@/components/layouts/SectionWrapper';
 import BlockRadioBox from '@/components/pages/application/form-components/BlockRadioBox';
 import SectionContent from '@/components/pages/application/SectionContent';
 import SectionFooter from '@/components/pages/application/SectionFooter';
 import SectionTitle from '@/components/pages/application/SectionTitle';
 import { ApplicationOutletContext } from '@/global/types';
+import { useApplicationContext } from '@/providers/context/application/ApplicationContext';
+import { FileExtentionTypes } from '@pcgl-daco/data-model';
 
 const { Text } = Typography;
 const { useToken } = theme;
 
 const rule = createSchemaFieldRule(ethicsSchema);
 
-enum AllowedFilesEnum {
-	PDF = 'application/pdf',
-	DOC = 'application/msword',
-	DOCX = 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
-}
-
```

Comment:
Testing locally and I believe the copy still includes: `Allowed file types: pdf, doc, docx,`